frontend/src/components/VisitRouter/VisitTable.js

```diff
@@ -0,0 +1,30 @@
+import React from "react"
+import { Link } from "react-router-dom"
+import Grid from "@material-ui/core/Grid"
+import Container from "@material-ui/core/Container"
+import { PrevPointCopy } from "../Typography"
+import PrevPointButton from "../PrevPointButton"
+
+export default function VisitTable() {
```

Comment:
Im kind of curious about a component with no props. why use this?

Reply:
Its just a stub, setting up for props to be a participants list of visits. that  api endpoint doesnt  exist yet.